### ISSUE TITLE
[4.6.13] PowerShell error after upgrading

### ISSUE THREAD

**Issue by dhirschfeld** [conda/conda#8571]
After upgrading `4.6.8 -> 4.6.13` I'm now seeing an error about `Add-CondaEnvironmentToPrompt` when I start powershell
```powershell
Add-CondaEnvironmentToPrompt : The term 'Add-CondaEnvironmentToPrompt' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:10 char:1
+ Add-CondaEnvironmentToPrompt
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Add-CondaEnvironmentToPrompt:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

**Comment by dhirschfeld** [comment]
This issue is just a placeholder/reminder until I can find time to investigate further...

**Comment by msarahan** [comment]
Already fixed in #8570

4.6.14 is on canary with this fix

**Comment by dhirschfeld** [comment]
Since it has already been addressed I'll close this issue...
